doc/api-extensions.md

```diff
@@ -2535,3 +2535,8 @@ Adds a new {config:option}`device-unix-hotplug-device-conf:ownership.inherit` co
 ## `unix_device_hotplug_subsystem_device_option`
 
 Adds a new {config:option}`device-unix-hotplug-device-conf:subsystem` configuration option for `unix-hotplug` devices. This adds support for detecting `unix-hotplug` devices by subsystem, and can be used in conjunction with {config:option}`device-unix-hotplug-device-conf:productid` and {config:option}`device-unix-hotplug-device-conf:vendorid`.
+
+## `storage_volume_rootfs_size`
+
+Introduces a `volatile.rootfs.size` key, previously only supported for images, for container and virtual machine volumes that is used to store that volume's current size at any point.
```

Comment:
Should this be showing up in one of the reference documents that are 
```suggestion
Introduces a `volatile.rootfs.size` key, previously only supported for images, for container and virtual machine volumes. This key is used to store a volume's current size at any point.
```

Reply:
Clarifying that the clause following "virtual machine volumes that ..." is meant to refer to `key` and not `volumes`. 

Question: Should the `volatile.rootfs.size` key show up in a reference page somewhere? I could not find it in the PR's RTD build. For example, should it show up [here](https://canonical-ubuntu-documentation-library--14595.com.readthedocs.build/lxd/en/14595/config-options/)? Or in the storage volume configuration options like [here](https://canonical-ubuntu-documentation-library--14595.com.readthedocs.build/lxd/en/14595/reference/storage_zfs/)?

Reply:
> Clarifying that the clause following "virtual machine volumes that ..." is meant to refer to key and not volumes.

Yes, it is. Now I see how that is confusing,

> Question: Should the volatile.rootfs.size key show up in a reference page somewhere?

I always assumed it was a pattern to not include `volatile.*` keys in the docs as they are not intended (I think) to be set by users. Although for some reason it is apparentely allowed.
There seems to be a lack of pattern, the only `volatile.*` keys that are included are defined [here](https://github.com/hamistao/lxd/blob/disk_size_fixes/lxd/storage/utils.go#L495) and here(https://github.com/hamistao/lxd/blob/disk_size_fixes/lxd/instance/instancetype/instance.go#L111). Everywhere else they are do not follow a metadata doc comment and I do not see a pattern for why some of them are included and some aren't.